README.md

```diff
@@ -12,38 +12,62 @@ _Need help? Join [Weave Community Slack][slackjoin]._
 
 ## Installation
 
+`eksctl` is available to install from official releases as described below. While you may be able to find it via package managers (e.g. Homebrew, Chocolatey, Scoop), these packages are not produced nor managed by AWS. We recommend that you install `eksctl` from only the official GitHub releases. 
+
+### For Unix
 To download the latest release, run:
 
-```
-curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
-sudo mv /tmp/eksctl /usr/local/bin
-```
+```sh
+# for ARM systems, set ARCH to: armv6, armv7 or arm64
+ARCH=amd64
```

Comment:
looks like Github didn't send my comment or i missed something but here is a little suggestion
```suggestion
ARCH=$(uname -m) # or $(go env GOARCH)
```

Reply:
Nice one! The `$(uname -m)` gets expanded to `x86_64` on my `amd64` machines. I think we could go with the latter.